src/resources/webhook_events.rs

```diff
@@ -579,8 +578,7 @@ impl<'r> Signature<'r> {
             .collect();
         let t = headers.get("t").ok_or(WebhookError::BadSignature)?;
         let v1 = headers.get("v1").ok_or(WebhookError::BadSignature)?;
-        let v0 = headers.get("v0").map(|r| *r);
-        Ok(Signature { t: t.parse::<i64>().map_err(WebhookError::BadHeader)?, v1, v0 })
```

Comment:
Since `v0` was not required here, this change should not affect runtime behavior at all (including `v0` will still just be ignored as it is now, matching https://stripe.com/docs/webhooks/signatures#verify-manually)

Reply:
I think the purpose of versioning is in the case of a signing key leak and them having to permanently invalidate v1 (as they have "done" for v0 for testing purposes). Perhaps it would make sense to actually rework this to allow configuring the minimum-supported signature version at runtime without requiring a new release of the library...

In the mean time, I think it makes sense to get rid of this ignored-and-useless field.